packages/next/src/build/webpack/loaders/next-flight-loader/index.ts

```diff
@@ -55,11 +53,12 @@ export default async function transformSource(
 
     if (assumedSourceType === 'module') {
       if (clientRefs.includes('*')) {
-        return callback(
+        this.callback(
```

Comment:
If you're not adding `this.async()` you can also `return [code, map]` and `throw` for errors.